code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -244,6 +244,10 @@
 			if(overmind)
 				I.color = overmind.blob_reagent_datum.complementary_color
 			flick_overlay_view(I, src, 8)
+		if(factory && (locate(/obj/structure/blob/core) in range(2, src)))
+			adjustHealth(-maxHealth*0.1)
+		if(factory && (locate(/obj/structure/blob/node) in range(2, src)))
```

Comment:
>calling ```range(2, src)``` three times in the same life tick

HECK NO.

Reply:
Ok, i've moved it to a single call per life tick